### PR TITLE
[bitnami/mongodb-sharded] Add tests and publishing using VIB

### PR DESCRIPTION
**Pull request by FraPazGal**
Signed-off-by: FraPazGal <fdepaz@vmware.com>

### Description of the change

The main objective of this PR is to publish our Bitnami MongoDB Sharded Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/FraPazGal/charts/pull/108

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by agomezmoron — DISMISSED** [review]
LGTM! Nice contribution Fran!

File: .vib/mongodb-sharded/vib-publish.json [hunk 1]
```diff
@@ -36,6 +36,18 @@
...
+          "action_id": "goss",
+          "params": {
+            "resources": {
+              "path": "/.vib/mongodb-sharded/goss"
+            },
+            "remote": {
+              "workload": "sts-mongodb-sharded-mongos"
+            },
+            "vars_file": "vars.yaml"
+          }
```
Comment (joancafom) [inline .vib/mongodb-sharded/vib-publish.json:50]: You only included the GOSS action in the `vib-publish` workflow but forgot about `vib-verify`.

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your changes @FraPazGal ! The logic and tests LGTM, but you forgot to add the test case in the `vib-verify` pipeline so it will not trigger.

File: .vib/mongodb-sharded/vib-publish.json (hunk 1 above)
Comment (FraPazGal) [reply .vib/mongodb-sharded/vib-publish.json:50]: Damn, I totally missed this. Added the action and increased the timeout to some of the tests; of the 4 pipelines, 1 of them did timeout. Ref: [action run(with goss)](https://github.com/FraPazGal/charts/actions/runs/3089823682)

**Review by joancafom — APPROVED** [review]
LGTM!

**Pull request merged**